File: app/src/main/java/com/example/msimple/data/model/ICurrentModel.java
package com.example.msimple.data.model;

import com.example.msimple.data.vos.CategoryVO;
import com.example.msimple.data.vos.CurrentVO;

import java.util.List;

public interface ICurrentModel {

    void getCurrentProgram(Response response);


    interface Response{

        void onSuccess(CurrentVO currentVO);
        void onFail(String message);

    }
}

File: app/src/main/java/com/example/msimple/data/model/ICurrentModel.kt
package com.example.msimple.data.model

import com.example.msimple.data.vos.CategoryVO
import com.example.msimple.data.vos.CurrentVO

interface ICurrentModel {

    fun getCurrentProgram(response: Response)


    interface Response {

        fun onSuccess(currentVO: CurrentVO)
        fun onFail(message: String)

    }
}
